maintainers/scripts/gnome.sh

```diff
@@ -170,23 +170,23 @@ if [ "$action" == "update-all" ]; then
       echo "Skipping $project"
     else
       echo "= Updating $project to $majorVersion" >&2
-      update_project $project $majorVersion
+      update_project "$project" "$majorVersion"
       echo >&2
     fi
   done
 else
-  project="$2"
-  majorVersion="$3"
+  project=$2
```

Comment:
Well, because if you start caring about the quoting, it is bad style not to quote multi-word variable anywhere? Even if it happens to work in the specific case.